datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/event/README.md

```diff
@@ -0,0 +1,8 @@
+# Event Integration
+
+Choose this type of integration if your integration primarily sends events to Datadog.
+
+This template is for integrations that focus on event collection.
+
+There is no Python package for this type of integration, it essentially contains a manifest.
```

Comment:
```suggestion
There is no Python package for this type of integration, only a manifest is essential.
```

Reply:
Not sure I'm grasping the meaning of `it essentially contains a manifest`, so please let me know if this is inaccurate.